logverification/fetchmassif_not_test.go

```diff
@@ -0,0 +1,44 @@
+package logverification
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"github.com/datatrails/go-datatrails-common/azblob"
+	"github.com/datatrails/go-datatrails-common/logger"
+	"github.com/datatrails/go-datatrails-merklelog/massifs"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestFetchMassif(t *testing.T) {
```

Comment:
I can get rid of this, but archiving it somewhere might be useful if we need to do this in future

Reply:
ooo we should have this somewhere useful, maybe in slack or confluence readily available.

its really useful code

Reply:
but for sure i think this doesn't belong here?